plutus-core/plutus-ir/test/prettyprintingReadable/listMatch.golden

```diff
@@ -1,8 +1,13 @@
-letrec data (List_0 :: * -> *) a_1 | match_List_2 where
-         Nil_3 : List_0 a_1
-         Cons_4 : a_1 -> List_0 a_1 -> List_0 a_1
-in match_List_2
-     {all a_1. a_1 -> a_1} (Nil_3 {all a_1. a_1 -> a_1}) {all a_1. a_1 -> a_1}
-     (/\a_1 -> \(x_5 : a_1) -> x_5) (\(head_6 : all a_1. a_1 -> a_1) (tail_7
-                                      : List_0 (all a_1. a_1 -> a_1)) ->
-                                       head_6)
\ No newline at end of file
+letrec
+  data (List_0 :: * -> *) a_1 | match_List_2 where
+    Nil_3 : List_0 a_1
+    Cons_4 : a_1 -> List_0 a_1 -> List_0 a_1
+in
+match_List_2
```

Comment:
Is it better to add indentation here? That would make it more clear where the body of the binding ends (especially if there are multiple bindings, e.g. `let ... in let ... in let ... in`)

Reply:
I think the whole point is to avoid increasing the indentation when there are multiple let bindings! The existing prettyprinter does this already. I think in other cases where it's ambiguous we should get parentheses? :thinking:

Reply:
Ok, I'm fine either way. Since we have `LetMerge` the indentation shouldn't be terrible, but not having indentation isn't bad either.